exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,24 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80)
+  {
+    return "A";
+  }
+  else if (mark < 80 && mark > 60)
+  {
+    return "B";
+  }
```

Comment:
Nice. You read this part of the code problem well and correctly noted "<=60" should be in the else if statement.